scripts/build.sh

```diff
@@ -0,0 +1,83 @@
+npm install -g pkg
+
+yarn build
+
+yarn install
+
+# Remove type from package.json files
+sed -i '' '/"type": "module",/d' ./package.json
+cd packages && sed -i '' '/"type": "module",/d' ./*/package.json && cd ..
+
+echo "import { cli } from '@percy/cli';\
+$(cat ./packages/cli/dist/percy.js)" > ./packages/cli/dist/percy.js
+
+sed -i '' '/Inserts formatFilepath function/a \
+function formatFilepath(filepath) {\
+  let path = url.pathToFileURL(filepath).href.replace("file:///","");\
+  if (!path.includes("C:")) {\
+    path = "/" + path;\
+  }\
+  return path;\
+}
+' ./packages/cli/dist/commands.js
+
+sed -i '' 's/import(url.pathToFileURL(modulePath).href);/import(formatFilepath(modulePath));/g' ./packages/cli/dist/commands.js
+
+echo "import { execSync } from 'child_process';\
+$(cat ./packages/core/dist/install.js)" > ./packages/core/dist/install.js
+
+sed -i '' '/Update outdir to absolute path/a \
+      var output = execSync(command, { encoding: "utf-8" }).trim();\
+      archive = output.concat("/", archive);\
+      outdir = output.concat("/", outdir);
+' ./packages/core/dist/install.js
+
+sed -i '' '/let archive/a \
+  var command = "pwd";\
+  if (archive.includes("C:")) {\
+    command = "cd";\
+  }\
+  outdir = outdir.replace("C:\\\\","");\
+  archive = archive.replace("C:\\\\","");
+' ./packages/core/dist/install.js
+
+sed -i '' '/let outdir/a \
+  if (outdir.charAt(0) == "/") {\
+    outdir = outdir.replace("/", "");\
+  }
+' ./packages/core/dist/install.js
```

Comment:
can we please maintain this as part of the code itself behind an `ENV`?  this will grow unmaintainable in future.